client/packages/coldchain/src/Equipment/DetailView/Tabs/Summary.tsx

```diff
@@ -136,7 +153,36 @@ export const Summary = ({ draft, onChange }: SummaryProps) => {
         </Section>
         <Section heading={t('heading.cold-chain')}>
           <Row label={t('label.cold-storage-location')}>
-            <BasicTextInput value={''} disabled fullWidth />
+            {locations ? (
+              <Autocomplete
+                multiple
+                id="multi-select-location"
+                options={locations}
+                getOptionLabel={option => option.label}
+                defaultValue={defaultLocations}
+                filterSelectedOptions
+                onChange={(
+                  _event,
+                  newSelectedLocations: {
+                    label: string;
+                    value: string;
+                  }[]
+                ) => {
+                  onChange({
+                    locationIds: newSelectedLocations.map(
+                      location => location.value
+                    ),
+                  });
+                }}
+                renderInput={params => (
+                  <TextField
+                    {...params}
+                    label="Current Asset Locations"
```

Comment:
these need to be translations (though I think they should be removed.. see my suggestion branch!)